libmcount/internal.h

```diff
@@ -440,9 +447,11 @@ struct mcount_disasm_info {
 	struct sym		*sym;
 	unsigned long		addr;
 	unsigned char   	insns[64];
+	struct cond_branch_info cond_info[MAX_COND_BRANCH];
```

Comment:
I think "branch_info" is slightly better.

Reply:
For the struct tag or the variable ?

Reply:
The variable.